website/static/js/logFeed.js

```diff
@@ -173,6 +173,7 @@ var createLogs = function(logData){
             // The node type, either 'project' or 'component'
             // NOTE: This is NOT the component category (e.g. 'hypothesis')
             nodeType: item.node.is_registration ? 'registration': item.node.node_type,
+            projectUrl: item.node.node_type === 'project' ? '/' + item.node.registered_from_id + '/' : item.node.url,
```

Comment:
The following comment from an outdated diff still needs to be addressed:

> Also, please move projectUrl directly under nodeUrl (line 179), so that the two extremely similar pieces of information will be side-by-side for easy comparison.

Putting them together will reduce the risk of someone using the wrong one. Hopefully.